change/@ni-nimble-angular-4195d860-6f9d-49d1-83bc-7ccc1b77743d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can you update the banner spec / quick double check the API it shows is mostly up to date?

Reply:
Done